fuzz.py

```diff
@@ -5,10 +5,15 @@
 a coverage-guided fuzzer I'm working on.
 """
 
+import re
+
 import hypothesmith
 from hypothesis import HealthCheck, given, settings, strategies as st
 
 import black
+from blib2to3.pgen2.tokenize import (  # NB: This intends to (and currently does) import the black-vendored blib2to3
```

Comment:
I added this comment based on a memory of reading some code that variously imported either the `blib2to3` code or Python's built-in `lib2to3` code.  But after checking it again, I realize they use different module names, so I think this comment is redundant and can be removed.